src/test/java/gov/cancer/tests/crosscutting/RelatedResources_Test.java

```diff
@@ -0,0 +1,122 @@
+package gov.cancer.tests.crosscutting;
+
+import java.util.Iterator;
+import java.util.List;
+
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import gov.cancer.framework.ExcelDataReader;
+import gov.cancer.pageobject.crosscutting.RelatedResourcesPage;
+import gov.cancer.pageobject.helper.RelatedResource;
+import gov.cancer.tests.TestObjectBase;
+import gov.cancer.tests.TestRunner;
+/**
+* Test Case: 
+* Verify Related Resources section and links
+
+* Test scenarios:
+* Assert Related Resources section exists on the page
+* Assert Related Resources internal links exists on the page
+* Assert non-zero number of Related Resources internal links are present.
+* Assert all Related Resources internal links have non-blank hrefs.
+* Assert all Related Resources internal link hrefs contain no whitespace.  (i.e. no leading/trailing/embedded spaces, tabs, new lines.)
+* Assert all Related Resources external links have “Exit Disclaimer” styling 
+
+* Negative Tests:
+8 Assert no Related Resources section/links appear on the page */
+
+
+public class RelatedResources_Test extends TestObjectBase {
+
+  @Test(dataProvider = "getPagesWithRelatedResources")
+  public void verifyRelatedResourcesSectionAppears(String path) {
+
+	  // Get the page.
+	  TestRunner.run(RelatedResourcesPage.class, path, (RelatedResourcesPage page) -> {
```

Comment:
Fix indenting to align with the rest of the code.

Reply:
fixed